crmsh/sbd.py

```diff
@@ -144,14 +155,35 @@ def _get_sbd_device(self):
             dev_list = self._get_sbd_device_interactive()
         self._sbd_devices = dev_list
 
+    def _correct_sbd_watchdog_timeout_for_s390(self):
```

Comment:
Cosmetic feedback, ideally, you may consider to the better function names, for example, the same prefix "_adjust" or
"_update" for the following,

def _adjust_systemd()

def _update_configuration()
def update_configuration()  # these two are too close to tell the difference just from their names.

def _correct_sbd_watchdog_timeout_for_s390()
def _determine_stonith_watchdog_timeout()
def _determine_sbd_watchdog_timeout()